internals-js/src/specs/connectSpec.ts

```diff
@@ -130,14 +172,23 @@ export class ConnectSpecDefinition extends FeatureDefinition {
       input SourceHTTP {
         baseURL: String!
         headers: [HTTPHeaderMapping!]
+
+        # added in v0.2
+        path: JSONSelection
+        query: JSONSelection
       }
     */
-    const SourceHTTP = schema.addType(new InputObjectType(this.typeNameInSchema(schema, SOURCE_HTTP)!));
+    const SourceHTTP = schema.addType(
+      new InputObjectType(this.typeNameInSchema(schema, SOURCE_HTTP)!),
+    );
     SourceHTTP.addField(new InputFieldDefinition('baseURL')).type =
-      new NonNullType(schema.stringType());
+      schema.stringType();
```

Comment:
```suggestion
      new NonNullType(schema.stringType());
```

Until we have `origin`